### PR TITLE
Update OMF Ingress Documentation for v1

### PR DESCRIPTION
**Pull request by dleeatosisoft**

### CODE REVIEW

File: Documentation/DataIngress/Data_Ingress.md [hunk 1]
```diff
@@ -6,10 +6,9 @@ Data Ingress and OSIsoft Message Format
 =======================================
 
 You use OSIsoft Message Format (OMF) to achieve high-throughput asynchronous data ingress 
-into the OCS data store. The following terms might be useful for understanding the information
-in this and subsequent topics:
+into the Sequential Data Store. The following terms might be useful for understanding the information
+in this and subsequent sections:
```
Comment (osimarkrogers) [inline Documentation/DataIngress/Data_Ingress.md]: I would say "the following terms are useful"

File: Documentation/DataIngress/Data_Ingress.md [hunk 2]
```diff
@@ -6,10 +6,9 @@ Data Ingress and OSIsoft Message Format
...
 
-* A producer of OMF messages needs a *client* identity to authenticate and send messges.
+* A producer of OMF messages needs a *Client* identity to authenticate and send messges.
```
Comment (osimarkrogers) [inline Documentation/DataIngress/Data_Ingress.md]: typo in "messages" (messges)

File: Documentation/DataIngress/OMF_Ingress_Specification.md [hunk 3]
```diff
@@ -47,16 +47,16 @@ OMF message types fall into three categories: Type, Container, and Data, which a
...
 * **Span Type**
 
-  Span Types are not supported in OCS Data Store.
+  Span Types are not supported in Sequential Data Store.
 
 * **Property Types and Formats**
 
   OMF supports setting the ``format`` keyword to specify how a particular JSON type should 
-  be interpreted. The following is a mapping for the OCS Data Store supported 
+  be interpreted. The following is a mapping for the Sequential Data Store supported
```
Comment (osimarkrogers) [inline Documentation/DataIngress/OMF_Ingress_Specification.md]: Add colon at end of last sentence.

File: Documentation/DataIngress/OMF_Ingress_Specification.md [hunk 4]
```diff
@@ -81,7 +81,7 @@ string   | date-time | DateTime
 Container messages
 ------------------
 
-A Container message is interpreted as a SdsStream in the OCS Data Store. The keywords 
+A Container message is interpreted as a SdsStream in the Sequential Data Store. The keywords
```
Comment (osimarkrogers) [inline Documentation/DataIngress/OMF_Ingress_Specification.md]: Should be "an SdsStream".

File: Documentation/DataIngress/Data_Ingress.md (hunk 1 above)
Comment (dleeatosisoft) [reply Documentation/DataIngress/Data_Ingress.md]: Fixed

File: Documentation/DataIngress/Data_Ingress.md (hunk 2 above)
Comment (dleeatosisoft) [reply Documentation/DataIngress/Data_Ingress.md]: Fixed.

File: Documentation/DataIngress/OMF_Ingress_Specification.md (hunk 3 above)
Comment (dleeatosisoft) [reply Documentation/DataIngress/OMF_Ingress_Specification.md]: Fixed

File: Documentation/DataIngress/OMF_Ingress_Specification.md (hunk 4 above)
Comment (dleeatosisoft) [reply Documentation/DataIngress/OMF_Ingress_Specification.md]: Fixed.

File: Documentation/DataIngress/OMF_Ingress_Topics.md [hunk 5]
```diff
@@ -5,16 +5,16 @@ uid: omfIngressTopics
...
+A Topic is used to aggregate data received from Clients. The Topic then makes the data available for consumption 
+via a Subscription. A Topic must contain at least one Client Id. Clients may be added to or 
+removed from an existing Topic. A given Client may also belong to multiple Topics in separate namespaces. 
 
-When a topic is created, data sent from its assigned clients is routed to a special queue 
-where it can be consumed by a subscription. This queue provides a buffer of up to one day for 
-subscriptions which are temporarily unable to receive data.
+When a Topic is created, data sent from its associated Clients is routed to a special queue 
+where it can be consumed by a Subscription. This queue provides a buffer of up to one day for 
+Subscriptions which are temporarily unable to receive data.
```
Comment (osimarkrogers) [inline Documentation/DataIngress/OMF_Ingress_Topics.md]: Add comma after "Subscriptions" on line 17.

File: Documentation/DataIngress/OMF_Ingress_to_OCS.md [hunk 6]
```diff
@@ -2,14 +2,13 @@
...
 ---
 
-Data ingress to Cloud Services using OMF
+Data Ingress to Cloud Services using OMF
 ========================================
 
 You can use OSIsoft Message Format (OMF) to achieve high-throughput asynchronous data ingress 
-into the OCS data store. The following terms might be useful for understanding the information
-in this and subsequent topics:
+into the Sequential Data Store. The following terms might be useful for understanding the information
```
Comment (osimarkrogers) [inline Documentation/DataIngress/OMF_Ingress_to_OCS.md]: "following terms are useful"

Comment (dleeatosisoft) [reply Documentation/DataIngress/OMF_Ingress_to_OCS.md]: Fixed.

**Pull request merged**